### PR TITLE
23/10/2023

### PR DESCRIPTION
**Pull request by xlassix**
- Removed Axios
- Added husky

### CODE REVIEW

File: src/methods/generate/index.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const generate = async (threshold = 3, keyCount = 5): Promise<GeneratedKe
   */
 
   // other members of the array ingredients used in the algorithm
-  for (let i = 0; i < threshold; i++) {
+  for (let i = 0; i < threshold - 1; i++) {
```
Comment (ravish1729) [inline src/methods/generate/index.ts]: why -1??

File: src/methods/transferOwnership/index.ts [hunk 2]
```diff
@@ -11,47 +9,37 @@ export const transferOwnership = async (
...
+          resetSharedTo,
+        });
       })
     );
     return {
       isSuccess: isEqual(...data) && data[0]?.message === "success",
       error: null,
     };
   } catch (err: any) {
+    console.log({ err });
```
Comment (ravish1729) [inline src/methods/transferOwnership/index.ts:39]: console remove

**Review by ravish1729 — APPROVED** [review]
LGTM

File: src/methods/generate/index.ts (hunk 1 above)
Comment (xlassix) [reply src/methods/generate/index.ts]: Issue with 3/5 keys when a known key is provided

**Pull request merged**